CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # @digitalbazaar/cborld ChangeLog
 
+## 8.0.0 - 2024-mm-dd
+
+### Added
+- Add TypedLiteral, UntypedLiteral encoders and decoders to more generally
+  handle type-specific behavior such as cryptosuite string encoding as well
+  as pass through uncompressible untyped values.
+
+### CHANGED
+- Restructure term registry system to be more general with four tables which
+  can all be passed by the user during encoding and decoding:
+    - keywordsTable: for JSON-LD keywords (e.g. '@context', '@type')
```

Comment:
Why can this be passed by the user? I recommend we don't expose this one at this time.